diff --git a/build/modules/vc/data-integrity/libraries/nativeDocumentLoader.mjs b/build/modules/vc/data-integrity/libraries/nativeDocumentLoader.mjs
index 49ee03fb80aa045daee6fd115548d7e41235ad82..aa7e43a80ff23848e5ba7e51e4c1b3945bf7da15 100644
--- a/build/modules/vc/data-integrity/libraries/nativeDocumentLoader.mjs
+++ b/build/modules/vc/data-integrity/libraries/nativeDocumentLoader.mjs
@@ -2,7 +2,7 @@
 
 //#region src/modules/vc/data-integrity/libraries/nativeDocumentLoader.ts
 async function getNativeDocumentLoader() {
-	const loader = await import("@digitalcredentials/jsonld/lib/documentLoaders/node");
+	const loader = await import("@digitalcredentials/jsonld/lib/documentLoaders/node.js");
 	if (!loader) throw new Error("Could not load node document loader. Module did not contain a loader function");
 	if (typeof loader === "function") return loader;
 	if (typeof loader === "object" && typeof loader.default === "function") return loader.default;
