diff --git a/build/modules/vc/data-integrity/libraries/contexts/defaultContexts.js b/build/modules/vc/data-integrity/libraries/contexts/defaultContexts.js
index e6ff8ba29468b1084293dc00d76204d326713d6b..ae96bc18f25cc3ca3522e5ec71403fc46dd704a4 100644
--- a/build/modules/vc/data-integrity/libraries/contexts/defaultContexts.js
+++ b/build/modules/vc/data-integrity/libraries/contexts/defaultContexts.js
@@ -7,6 +7,7 @@ const credentials_v1_1 = require("./credentials_v1");
 const dataIntegrity_v2_1 = require("./dataIntegrity_v2");
 const did_v1_1 = require("./did_v1");
 const ed25519_v1_1 = require("./ed25519_v1");
+const linkedVp_v1 = require("./linkedvp_v1");
 const odrl_1 = require("./odrl");
 const purl_ob_v3po_1 = require("./purl_ob_v3po");
 const schema_org_1 = require("./schema_org");
@@ -33,5 +34,6 @@ exports.DEFAULT_CONTEXTS = {
     'https://purl.imsglobal.org/spec/ob/v3p0/context.json': purl_ob_v3po_1.PURL_OB_V3P0,
     'https://w3c-ccg.github.io/vc-status-rl-2020/contexts/vc-revocation-list-2020/v1.jsonld': vc_revocation_list_2020_1.VC_REVOCATION_LIST_2020,
     'https://w3id.org/security/data-integrity/v2': dataIntegrity_v2_1.DATA_INTEGRITY_V2,
+    'https://identity.foundation/linked-vp/contexts/v1': linkedVp_v1.LINKEDVP_V1,
 };
 //# sourceMappingURL=defaultContexts.js.map
\ No newline at end of file
diff --git a/build/modules/vc/data-integrity/libraries/contexts/linkedvp_v1.js b/build/modules/vc/data-integrity/libraries/contexts/linkedvp_v1.js
new file mode 100644
index 0000000000000000000000000000000000000000..aba082dad86a1698e4e9a1d6415fe7d694b3579e
--- /dev/null
+++ b/build/modules/vc/data-integrity/libraries/contexts/linkedvp_v1.js
@@ -0,0 +1,13 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.LINKEDVP_V1 = void 0;
+exports.LINKEDVP_V1 = {
+    '@context': [
+    {
+      '@version': 1.1,
+      '@protected': true,
+      LinkedVerifiablePresentation: "https://identity.foundation/linked-vp/contexts/v1#LinkedVerifiablePresentation"
+    }
+  ],
+};
+//# sourceMappingURL=linkedvp_v1.js.map
\ No newline at end of file
diff --git a/build/modules/vc/data-integrity/libraries/contexts/linkedvp_v1.js.map b/build/modules/vc/data-integrity/libraries/contexts/linkedvp_v1.js.map
new file mode 100644
index 0000000000000000000000000000000000000000..da1e19557d08a98dc34db7382709b88f6dc15b19
--- /dev/null
+++ b/build/modules/vc/data-integrity/libraries/contexts/linkedvp_v1.js.map
@@ -0,0 +1,7 @@
+{
+  "version": 3,
+  "sources": ["node_modules/@credo-ts/core/build/modules/vc/data-integrity/libraries/contexts/linkedvp_v1.js"],
+  "sourcesContent": ["\"use strict\";\nObject.defineProperty(exports, \"__esModule\", { value: true });\nexports.LINKEDVP_V1 = void 0;\nexports.LINKEDVP_V1 = {\n    '@context': [\n    {\n      '@version': 1.1,\n      '@protected': true,\n      LinkedVerifiablePresentation: \"https://identity.foundation/linked-vp/contexts/v1#LinkedVerifiablePresentation\"\n    }\n  ],\n};\n//# sourceMappingURL=linkedvp_v1.js.map"],
+  "mappings": ";AACA,OAAO,eAAe,SAAS,cAAc,EAAE,OAAO,KAAK,CAAC;AAC5D,QAAQ,cAAc;AACtB,QAAQ,cAAc;AAAA,EAClB,YAAY;AAAA,IACZ;AAAA,MACE,YAAY;AAAA,MACZ,cAAc;AAAA,MACd,8BAA8B;AAAA,IAChC;AAAA,EACF;AACF;",
+  "names": []
+}
