diff --git a/dist/cjs/index.cjs b/dist/cjs/index.cjs
index 89e9f6b79e5348cad8edcb700902f661877d6b26..e345e42be6dc26d99f14f2710447d5dc8e86f6be 100644
--- a/dist/cjs/index.cjs
+++ b/dist/cjs/index.cjs
@@ -802,7 +802,7 @@ var getBaseUrl = (id) => {
   const protocol = remainder.includes("localhost") ? "http" : "https";
   const [hostPart, ...pathParts] = remainder.split("/");
   let [host, port] = decodeURIComponent(hostPart).split(":");
-  host = host.split(".").map((label) => toASCII(label.normalize("NFC"))).join(".");
+  host = toASCII(host.normalize("NFC"));
   const normalizedHost = port ? `${host}:${port}` : host;
   const path = pathParts.join("/");
   return `${protocol}://${normalizedHost}${path ? "/" + path : ""}`;
